pytest_splunk_addon/standard_lib/fields_tests/test_templates.py

```diff
@@ -264,3 +264,53 @@ def test_eventtype(
             f"No result found for the search.\nsearch={search}\n"
             f"interval={splunk_search_util.search_interval}, retries={splunk_search_util.search_retry}"
         )
+
+
+    @pytest.mark.splunk_searchtime_fields
+    @pytest.mark.splunk_searchtime_fields_savedsearches
+    def test_savedsearches(
+        self,
+        splunk_search_util,
+        splunk_ingest_data,
+        splunk_setup,
+        splunk_searchtime_fields_savedsearches,
+        record_property,
+        caplog,
+    ):
+        """
+        Tests if all savedsearches in savedsearches.conf are generated in Splunk.
```

Comment:
update the docstring. we are not checking savedsearches are generated. we are testing each saved search is being executed properly and producing results.

Reply:
Done.